common/hooks/post-install/06-strip-and-debug-pkgs.sh

```diff
@@ -65,11 +65,10 @@ hook() {
 
 	STRIPCMD=/usr/bin/$STRIP
 
-	find ${PKGDESTDIR} -type f | while read f; do
-		if [[ $f =~ ^${PKGDESTDIR}/usr/lib/debug/ ]]; then
-			continue
-		fi
-
+	find ${PKGDESTDIR} \
+		-path ${PKGDESTDIR}/usr/share/ -prune -path ${PKGDESTDIR}/usr/lib/debug/ -prune \
```

Comment:
IMO, we should only consider files inside `/usr/bin` `/usr/lib` and `/usr/libexec`

Reply:
There's also `/opt` and the directories for all the cross toolchains.

Reply:
Since the only place where we guarantee no ELFs can be found is `/usr/share`, that's the only one I excluded

Reply:
binaries in /etc would be incorrect, too, but /etc contains very few files, so that's fine. I don't think we should strip `/opt`.

Reply:
We could maybe expand elf-in-usrshare to other directories, including `/etc`.

On the matter of `/opt`, it's fair to assume most third party stuff will be stripped already, but maybe stuff liked `rpi-userland` still needs to be?

Reply:
Confirmed, we have a `rpi-userland-dbg` packge.

Reply:
I will leave the decision regarding where we can have binaries for later. For now, this will just keep the old behavior.